mep/people/tests/test_views.py

```diff
@@ -165,7 +165,7 @@ def test_person_autocomplete(self):
         data = json.loads(result.content.decode('utf-8'))
         text = data['results'][0]['text']
         expected = ('<strong>Ms. Sylvia</strong> sylv.a <br />'
-                    'Subscription (1971-01-01 - 1971-01-31)')
+                    'Subscription (1971-01-01–1971-01-31)')
```

Comment:
I don't remember where we're using these, but I think this is unintelligible without spaces. Too confusing with the hyphens in the dates.